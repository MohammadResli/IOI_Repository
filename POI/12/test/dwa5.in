50
39 5 19 28 34 32 37 45 11 43 31 12 41 39 23 22 50 49 44 13 33 40 25 7 36 20 47 16 18 27 17 24 21 30 35 15 4 10 42 29
37 46 36 31 40 9 15 17 20 11 8 24 22 49 48 33 4 30 18 50 44 23 37 29 45 19 21 10 42 14 41 27 5 32 28 3 47 35
35 18 43 10 37 34 15 32 50 2 47 25 49 40 12 48 26 36 6 46 8 35 29 17 45 19 11 38 23 14 22 9 28 30 42 27
42 6 32 20 30 34 48 23 45 5 25 29 44 49 16 2 10 1 14 43 41 47 7 9 37 8 19 38 15 35 26 24 42 31 12 39 46 11 18 40 17 28 21
38 49 28 4 26 16 43 21 10 18 15 11 1 33 40 13 6 8 17 23 22 48 2 29 20 34 38 7 12 30 42 39 19 44 24 14 46 25 32
44 50 12 44 42 20 41 3 30 7 49 25 31 32 38 40 33 39 27 10 35 28 24 16 9 18 15 11 23 26 17 21 4 5 48 47 8 13 22 46 29 43 34 19 37
34 45 32 8 34 26 31 10 35 28 22 36 4 42 21 11 19 1 20 38 41 33 47 49 44 50 43 37 40 25 46 6 5 13 14
36 2 29 26 20 3 13 16 6 39 27 9 17 11 37 33 12 38 43 22 44 5 23 10 49 7 21 19 36 30 18 41 46 15 4 28 24
40 10 46 49 39 25 18 48 6 41 19 12 24 14 8 38 20 13 31 50 29 21 36 45 43 35 37 22 44 3 23 15 4 40 2 33 26 17 34 30 32
39 42 46 17 39 50 33 19 3 12 8 16 36 25 26 40 13 24 21 14 1 5 27 45 4 30 48 18 7 15 34 6 49 9 47 22 44 23 32 2
41 32 16 27 18 33 42 30 14 47 8 38 26 37 40 7 34 44 4 2 46 29 45 25 12 15 22 5 48 49 31 41 24 50 6 39 3 35 23 17 13 1
37 28 6 46 38 13 49 33 19 9 26 10 40 42 21 50 27 4 20 41 8 3 31 32 11 30 34 44 24 1 5 48 45 25 47 36 15 29
37 16 36 46 7 22 40 49 33 34 8 26 5 41 37 30 28 20 11 38 24 43 27 48 12 42 47 32 6 17 1 14 9 18 23 21 10 45
39 25 40 39 20 4 32 42 45 38 28 21 50 5 47 17 41 27 33 18 23 46 34 22 11 48 9 3 35 10 7 16 13 26 43 2 24 29 30 44
37 6 11 4 40 21 1 9 10 31 44 37 23 32 27 49 26 3 24 41 17 48 8 29 16 50 30 25 5 45 47 38 39 43 46 20 12 2
37 21 46 14 18 23 48 24 50 38 1 31 42 47 44 33 27 43 39 26 20 5 11 22 15 4 30 32 41 13 37 29 34 8 49 6 35 10
38 11 5 4 10 6 49 42 8 50 27 19 24 28 25 46 15 29 33 38 34 13 36 41 31 2 1 3 39 21 22 45 14 18 20 23 48 47 9
41 38 35 43 42 41 39 20 48 49 1 47 17 10 45 27 29 32 3 2 46 30 6 21 23 19 24 28 34 40 50 16 11 36 9 13 33 8 5 25 4 14
37 36 34 3 33 24 30 5 4 10 1 43 28 26 2 7 12 6 9 27 42 38 48 8 46 18 23 49 22 44 17 47 32 41 45 25 37 35
39 26 5 4 12 16 22 25 40 48 15 21 23 2 44 35 38 1 32 9 45 14 27 13 17 46 39 30 18 7 37 28 6 42 36 47 41 34 33 8
45 29 4 26 28 20 37 41 16 24 39 31 18 34 43 9 17 45 22 15 13 6 49 5 8 32 1 40 25 10 36 50 7 46 33 2 47 35 12 23 42 30 14 38 44 48
38 49 35 27 3 10 21 1 37 25 41 14 17 11 43 31 13 9 34 39 47 33 23 38 44 45 16 40 28 5 2 19 42 20 50 8 7 6 46
38 38 10 48 36 1 3 28 2 17 6 42 4 47 20 11 27 8 35 13 14 46 18 34 15 40 5 25 39 41 21 22 19 31 16 32 26 9 30
34 14 46 37 12 17 5 34 18 10 13 2 9 6 16 4 8 48 49 39 35 28 11 38 40 15 29 19 30 21 26 32 1 43 47
35 17 28 31 9 7 4 43 50 3 33 15 29 22 40 18 37 20 6 48 10 23 5 11 12 21 47 30 42 41 19 1 46 26 45 14
40 31 12 20 30 23 21 19 45 13 5 15 37 24 46 49 38 9 16 42 34 33 14 10 28 7 4 3 41 50 40 39 25 8 35 11 29 27 43 6 48
35 2 26 16 49 47 6 48 8 42 40 10 33 30 19 17 36 38 13 3 18 14 29 23 11 45 43 12 32 50 22 1 31 15 20 28
39 30 18 7 20 24 13 38 37 43 35 2 49 27 29 39 14 1 22 21 4 17 23 50 5 40 48 26 36 8 31 6 45 41 12 25 19 46 3 47
37 2 4 5 46 35 9 40 49 41 24 21 45 11 18 28 25 12 44 1 26 33 15 37 39 17 47 14 6 16 43 38 32 50 48 27 8 3
41 48 26 15 27 13 43 1 18 47 16 21 23 45 4 5 10 19 28 41 25 14 38 20 2 31 49 11 36 24 9 3 6 46 50 32 37 8 35 12 40 39
34 22 2 1 50 47 42 45 40 25 21 23 43 38 15 27 12 9 48 49 6 30 17 34 37 4 26 16 28 39 11 46 41 36 7
41 35 20 11 46 19 44 36 10 14 47 39 49 5 6 2 43 48 38 27 12 9 50 16 15 7 18 33 45 24 21 3 4 42 30 1 29 41 40 23 37 13
36 29 26 21 12 7 40 36 39 34 43 2 5 22 48 9 17 25 47 35 18 14 41 16 13 27 50 6 32 8 11 49 10 20 19 1 42
37 43 40 9 33 11 47 17 22 18 19 6 14 1 20 24 7 26 39 4 44 50 21 16 5 23 12 35 31 36 42 10 49 3 46 38 48 13
36 16 14 39 9 48 24 26 47 1 45 23 49 11 6 30 37 2 32 41 38 29 28 22 18 7 33 46 4 19 34 3 40 50 43 20 21
34 38 12 21 33 3 7 10 30 32 27 19 42 43 20 23 17 37 9 8 44 13 39 45 1 40 18 46 28 2 48 34 50 31 49
37 26 44 24 31 29 13 6 20 9 25 42 1 49 30 8 3 50 15 45 48 36 38 35 11 39 19 32 41 4 16 47 7 28 43 21 22 2
42 9 47 21 20 12 42 50 30 31 18 39 11 32 13 16 36 48 23 7 41 8 35 6 22 43 40 4 29 15 14 34 26 49 28 37 24 45 27 5 17 19 3
37 41 23 16 29 21 4 45 46 10 22 38 8 9 48 44 11 34 18 1 24 14 32 30 37 40 47 35 33 5 31 6 28 36 15 26 17 20
42 50 13 27 36 38 4 6 33 2 34 18 35 5 1 45 10 31 49 11 39 25 7 12 42 14 21 28 29 9 44 20 15 24 43 3 26 23 22 46 48 32 30
37 6 9 37 8 15 38 4 23 20 31 39 17 48 2 14 13 42 16 25 11 44 12 21 46 29 1 19 33 35 30 28 7 18 32 26 22 45
39 12 16 25 36 23 34 19 27 32 3 26 11 4 44 18 13 46 43 10 20 40 1 14 41 45 33 31 38 37 50 49 7 17 22 6 2 47 21 5
38 22 45 8 29 33 49 34 31 16 19 30 37 26 40 14 7 3 47 50 28 1 42 44 4 35 24 15 6 18 9 25 27 32 38 36 21 5 13
33 10 14 6 1 2 8 48 46 21 4 36 50 43 19 15 34 16 45 37 42 9 5 20 40 22 32 29 47 11 12 7 41 39
38 12 35 40 32 21 31 37 15 17 43 10 44 19 30 25 9 13 2 36 26 39 42 20 4 27 11 14 28 48 41 3 1 22 18 7 38 50 29
41 23 8 17 28 29 44 35 39 31 4 36 10 21 14 49 19 22 26 40 42 48 18 16 12 24 7 5 11 34 15 30 25 2 20 50 13 32 6 3 41 9
40 23 27 6 42 33 50 30 34 48 22 1 35 4 37 32 17 13 18 3 7 11 29 2 19 24 38 39 10 28 14 15 16 21 44 31 43 25 49 12 20
43 5 40 28 25 31 16 49 17 24 29 2 50 47 26 14 36 38 11 12 13 18 10 44 37 6 46 23 20 33 35 3 27 21 15 45 32 34 19 9 39 4 41 30
40 22 12 6 31 27 8 17 11 47 46 35 34 18 5 19 42 24 26 1 28 37 36 21 7 43 40 2 4 48 16 3 30 10 32 13 9 15 33 38 29
38 43 36 35 15 37 47 18 40 16 1 28 26 9 29 45 2 38 46 3 10 33 7 34 11 48 12 22 32 27 6 14 42 21 30 25 44 31 17
