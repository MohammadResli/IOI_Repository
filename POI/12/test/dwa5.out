31
1 2 3 6 7 8 9 11 14 17 18 19 21 24 26 27 28 29 30 31 32 34 35 38 42 44 45 46 48 49 50 
